lca-chimevc-stack/cloudformation-templates/chime-vc-siprec.yaml

```diff
@@ -21,11 +22,7 @@ Parameters:
     Type: CommaDelimitedList
     Description: Comma Delimited list of CIDR blocks that will be allow listed to send traffic to the Chime Voice Connector.  CIDR blocks must be between /27 and /32 in length and use public IP ranges.  Example 203.0.113.128/27, 198.51.100.76/32
 
-Conditions:
```

Comment:
Curioius why move 'Conditions' to near the end of the template.. Doesn't matter really since template is not ordered, except I think it's clearer to read to have them defined before they are used in 'Resources' section.